### PR TITLE
[FIX] odoo_sh: change img who explain storage space

### PR DESCRIPTION
**Pull request by roussieau**
The image that explains the storage in odoo.sh is outdated.

https://www.odoo.com/web#id=2948233&cids=1&menu_id=4720&action=333&active_id=1105&model=project.task&view_type=form

### CODE REVIEW

**Review by amigrave — APPROVED** [review]

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Comment by jcs-odoo** [comment]
@amigrave if you approve on behalf of sh-doc-review, you have to add the next reviewer after your approval. Usually, that's odoo/doc-review so they can merge the PR. 
Thanks :)

**Comment by jcs-odoo** [comment]
@robodoo retry

**Pull request closed without merging**
